### DISCUSSION TITLE
MiniMessage & Italic Components

### DISCUSSION THREAD

**Discussion by UnfamousThomas** [Minestom/Minestom#2320] [Help] [answered]
Hello.

I have a question related to italic item names. I know that all custom item names are italic by default, and you need to do:
`decoration(TextDecoration.ITALIC, false)` to disable them.

However, I am using MiniMessage format (made by same people who made Adventure), e.g: `<green>Rank has been changed successfully.</green>`, which also has <italic> tag. It deserializes the String into a Component, but my issue is the decoration.

Since I do component.decoration(TextDecoration, boolean), it means that it would remove any italics that were in the original string. I could potentially do something like `string.contains("<italic>")` but that is not my favourite solution either, as it means that the whole thing is italic, but I could potentially only want to have a word in the String as italic, e.g:
`<green>Rank has been changed <italic>successfully</italic>.</green>`.

Is there any valid solution for this? If not, this seems like a major issue with default italic custom names.

**Comment by Thorinwasher** [comment, marked as answer]
Try:
```xml
<!italic>My text
```

**Reply by UnfamousThomas** [reply]
That appears to work! Thank you.

**Reply by Nowaha** [reply]
It's also possible to do it shorthand, `<!i>`
